syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Tser {
    int32 user_id = 1;
    string username = 2;
    string email = 3;
    string password_hash = 4;
    google.protobuf.Timestamp created_at = 5;
    google.protobuf.Timestamp updated_at = 6;
    
}
